src/components/layout.css

```diff
@@ -9,7 +9,7 @@ html {
 a {
   color: rgb(183,191,16);
   background-color: transparent;
-  text-decoration: underline;
```

Comment:
The link underlines are an accessibility feature so I am reluctant to make this change.

Reply:
If I'm remembering waaay back to what I might have learned from Accessible Futures, I think as long as we make the text a different color, specifically and particularly in large blocks of text, I believe they still read as links -- the green is pretty differentiated. Also, my understanding is that many of the screen-readers are reading/scanning the actual code for links, so my counter proposal would be make sure that there are `title` and other screen-reader friendly attributes rather than relying on an underline indicator for accessibility. 

I think the locations where I would strongly prefer to not have underlines are the headers. For example, on the posts, I don't think the title needs and underline if there's a clear "Continue Reading" link.  It seems possible to find a way to have both clean design, but also have it be accessible.

Reply:
@kirstenkeister The current standard is that color along is not sufficient to distinguish links: https://www.a11yproject.com/checklist/#ensure-that-links-are-recognizable-as-links

Reply:
Okay. I'll work on applying this standard to the links in the main content.